content/02.body.md

```diff
@@ -2,13 +2,71 @@
+Figure {@fig:degree-bias} illustrates the problem of degree bias by showing node degree in separate networks for the same type of data.
+Depending on the methods by which the represented data were generated, networks of the same type of data may assign very different degree to the same nodes (Figure {@fig:degree-bias}B), and they may have overall degree distributions that differ greatly (Figure {@fig:degree-bias}A).
+The consequence of degree bias is that a difference in degree between two nodes may not reflect a difference in the number of true edges.
+Reliance on degree may be unfavorable depending on the prediction/analysis task being conducted and the magnitude of possible degree bias in the data.
+
+![**A.** Degree distributions of networks with and without degree bias can be very different.
+  Protein-protein interaction (PPI) literature-derived network from STRING [@doi:10.1093/nar/gky1131], high-throughput-derived networks from Rual et al. [@doi:10.1038/nature04209] and Rolland et al. [@doi:10.1016/j.cell.2014.10.050].
+  Transcription factor-target gene (TF-TG) literature-derived network from Han et al. [@doi:10.1093/nar/gkx1013] and high-throughput network from Lachmann et al. [@doi:10.1093/bioinformatics/btq466].
+  BioRxiv data from Rxivist [@doi:10.1101/515643; @doi:10.5281/zenodo.2566421] had edges split between those existing before and during/after 2018.
+  **B.** Uniform random sampling edges produces node degrees that are linearly correlated with those in the unsampled network.
+  70% of literature edges were sampled with uniform probability.
+  Systematically-derived and literature-derived networks do not produce linearly correlated node degree.
+  Assuming that the inclusion of an edge in the systematic network is independent of node degree, the systematic degree should be linearly correlated with the true number of relationships.
+  Edges found in literature-derived networks represent a non-random sample from systematically-derived edges.
+  Shown are the same PPI and TF-TG networks as in subfigure A.
+](https://github.com/greenelab/xswap-analysis/raw/5a257c9d28e0a4c3cc8f2fd8493f27ff4652f2c6/img/degree_bias.png){#fig:degree-bias width="100%"}
+
+Another reason why a reliance on degree can be unfavorable is that degree-based predictions may be nonspecific.
+Nonspecific predictions are based on information that, while potentially associated with true connections, is not reflective of the underlying causal relationships.
+For example, predicting that the highest degree node in a network shares edges with the remaining nodes to which it is not connected will often lead to many correct predictions, despite the fact that the predictions were generic to all nodes in the network.
+Gillis et al. [@doi:10.1371/journal.pone.0017258] explored the concept of prediction specificity with respect to gene function prediction, in which predictions based on gene multifunctionality could be, "potentially misleading with respect to causality."
```

Comment:
```suggestion
Gillis et al. [@doi:10.1371/journal.pone.0017258] examine prediction specificity in the context of gene function prediction and found that many predictions appear to rely primarily on multifunctionality and could be "potentially misleading with respect to causality."
```

Reply:
Agree with the suggestion!

Is past or present tense appropriate for describing previous work by others? The four verbs in this sentence ("examine", "found", "appear", "could be") are not all the same tense.